crates/loader/src/bindle/assets.rs

```diff
@@ -52,12 +55,12 @@ impl Copier {
     }
 
     async fn copy_all(&self, parcels: &[Label], dir: impl AsRef<Path>) -> Result<()> {
-        match future::join_all(parcels.iter().map(|p| self.copy(p, &dir)))
-            .await
-            .into_iter()
-            .filter_map(|r| r.err())
+        match stream::iter(parcels.iter().map(|p| self.copy(p, &dir)))
+            .buffer_unordered(MAX_PARALLEL_COPIES)
```

Comment:
@adamreese I _knew_ this was implemented _somewhere_!

Reply:
[StackOverflow FTW](https://stackoverflow.com/questions/43247212/join-futures-with-limited-concurrency)